html/changelogs/.all_changelog.yml

```diff
@@ -33707,9 +33707,9 @@ DO NOT EDIT THIS FILE BY HAND!  AUTOMATICALLY GENERATED BY ss13_genchangelog.py.
       mob.
   MacBlaze1:
   - tweak: Changed the eye damage of aiuri to .25 from 1, does not change healing.
-  - tweak: Changed toxin conversion of instabitaluri to 66% from 75%, does not buff
```

Comment:
Probably don't need to change the changelogs.

Reply:
Woops, I thought I double checked